packages/jni-swig-stub/realm.i

```diff
@@ -28,9 +28,7 @@ using namespace realm::jni_util;
     // Trigger loading of shared library when the swig wrapper is loaded
     static {
         // using https://developer.android.com/reference/java/lang/System#getProperties()
-        if (System.getProperty("java.specification.vendor").contains("Android")) {
-            System.loadLibrary("realmc");
-        } else {
+        if (!System.getProperty("java.specification.vendor").contains("Android")) {
```

Comment:
Nitpick: Maybe add a comment that we are loading the native library in the `RealmInitializer` on Android.